repository_service_tuf_worker/repository.py

```diff
@@ -707,16 +709,21 @@ def remove_targets(
         logging.debug(f"Delete targets. {result}")
         return asdict(result)
 
-    def bump_target_roles(self) -> bool:
+    def _run_online_roles_bump(self, force: Optional[bool] = False) -> bool:
         """
         Bumps version and expiration date of targets roles metadata
-        (`Targets` and `Succinct Delegated` targets roles).
+        (`Targets` and `Succinct Delegated` targets roles). The function also
+        updates 'snapshot' and 'timestamp'.
```

Comment:
Maybe it will be good to update the documentation so that the function doesn't focus on target roles specifically but instead underlines it updates all online roles:
```suggestion
        Bumps version and expiration date of all online roles (`Targets`,
        `Succinct Delegated` targets roles, `Timestamp` and `Snapshot`).
```